addon/session-stores/adaptive.js

```diff
@@ -133,6 +133,11 @@ export default Base.extend({
       const options = this.getProperties('cookieDomain', 'cookieName', 'cookieExpirationTime', 'cookiePath');
       options._fastboot = this.get('_fastboot');
       options._cookies = this.get('_cookies');
+
+      let expirationCookieName = `${this.get('cookieName')}-expiration_time`;
+      let expirationTime = parseInt(options._cookies.read(expirationCookieName), 10);
+      this.set('cookieExpirationTime', expirationTime);
```

Comment:
No, because the adaptive store's `cookieExpirationTime` property is already an alias for the underlying cookie store's `cookieExpirationTime` property anyway.